e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -20,7 +21,10 @@ Scenario('HMCTS super user removes an order from a case', async (I, caseViewPage
   removeOrderEventPage.addRemoveOrderReason('Entered incorrect order');
   await I.completeEvent('Submit');
   I.seeEventSubmissionConfirmation(config.superUserActions.removeOrder);
-  const defaultIssuedDate = new Date();
-
-  await orderFunctions.assertOrder(I, caseViewPage, order, defaultIssuedDate, true, true);
+  const issuedDate = new Date(2020, 4,26, 14, 33);
+  order = {...blankOrder, ...{
+    document: 'Blank order (C21).pdf',
+    reasonForRemoval: 'Entered incorrect order',
+  } };
```

Comment:
Maybe we could just change the fixture's document property to the above + add reasonforRemoval into the fixture also? I don't think this would affect any other test (maybe i am wrong :) )

Reply:
removal reason is fine to add however the document name that the system generates is different to the document name that is constructed in the case data used for this test

Reply:
ok sure :)

Reply:
done